app/src/main/java/android/coding/interview/makeitawesome/networking/ApiPhoto.java

```diff
@@ -0,0 +1,16 @@
+package android.coding.interview.makeitawesome.networking;
+
+import android.coding.interview.makeitawesome.model.Album;
+
+import java.util.List;
+
+import retrofit.http.GET;
+
+/**
+ * Created by antonio on 24/07/15.
+ */
+public interface ApiPhoto {
+
+    @GET("/")
```

Comment:
It should be @GET("/photo"), and the URL = "http://jsonplaceholder.typicode.com/

It's not super important, but it's something you won't very likely do like he did if you are used to work with REST APIs.